packages/argo-run/src/dev.ts

```diff
@@ -189,6 +189,9 @@ export async function dev(...args: string[]) {
                 extension.type === 'checkout'
                   ? extension.config.extensionPoints
                   : ['Checkout::PostPurchase::Render'],
+              config: extension.config
+                ? {metafields: extension.config.metafields}
+                : {},
```

Comment:
This will also return the metafields configuration for  a post purchase extension. I'm thinking we will probably use it in the future but I'm not sure if that's fine or not?

Also, is it okay to return a `config` object? I thought that this can be used for other configurations we might have but not sure if it's best to just have `metafields`?

Reply:
I don't have a strong opinion but I was imagining we'll want to access more things from the config in the future and it might be annoying to write a patch each time we need something.